### PR TITLE
B satish sutar patch 5

### PR DESCRIPTION
**Pull request by BSatishSutar**
## Check List (Check all the applicable boxes) 

- [ *] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ *] This PR does not contain plagiarized content.
- [ *] The title of my pull request is a short description of the requested changes.

## Screenshots

Removed twitter from Bio and added link to my new website/portfolio, so didn't add the screenshot. I hope that is ok.

## Note to reviewers

Removed 'Twitter' from my Bio and have added link to my new website/portfolio 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/8761"></a>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 [![Open in GitPod](https://img.shields.io/badge/Gitpod-Ready--to--Code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree) ![Uptime](https://img.shields.io/endpoint?url=https%3A%2F%2Fraw.githubusercontent.com%2FEddieHubCommunity%2Fmonitoring%2Fmaster%2Fapi%2Flink-free-linkfree-io%2Fuptime.json) [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT) [![GitHub release (latest by date)](https://img.shields.io/github/v/release/EddieHubCommunity/LinkFree)](https://github.com/EddieHubCommunity/LinkFree/releases) ![GitHub repo size](https://img.shields.io/github/repo-size/EddieHubCommunity/LinkFree)
 
+![2](https://github.com/BSatishSutar/LinkFree/assets/40925459/cd8b1c4d-9d4d-420d-8cc5-7d3e00c200ec)
+
+
```
Comment (SaraJaoude) [inline README.md:5]: As this is not part of your Profile I am removing this section in the README.md:

```suggestion
```

**Review by SaraJaoude — APPROVED** [review]
looks good

**Comment by SaraJaoude** [comment]
I have removed the banner you added to the LinkFree `README` me file as we are not looking to have this at the moment, particularly as it does not match the branding.  If you are proposing any changes to files (other than your own Profile) in this Repo then please follow the guidelines in the [Contributing Guide](https://github.com/EddieHubCommunity/LinkFree/blob/main/CONTRIBUTING.md).

**Comment by BSatishSutar** [comment]
> I have removed the banner you added to the LinkFree `README` me file as we are not looking to have this at the moment, particularly as it does not match the branding. If you are proposing any changes to files (other than your own Profile) in this Repo then please follow the guidelines in the [Contributing Guide](https://github.com/EddieHubCommunity/LinkFree/blob/main/CONTRIBUTING.md).

Sure. Noted. I replied on your reply to me for PR#8771. I won't edit the readme file henceforth.

**Pull request merged**